truffle/src/main/java/org/jruby/truffle/core/rope/ConcatRope.java

```diff
@@ -55,47 +56,6 @@ public byte getByteSlow(int index) {
         return right.getByteSlow(index - left.byteLength());
     }
 
-    @Override
-    @TruffleBoundary
-    public byte[] extractRange(int offset, int length) {
-        assert length <= this.byteLength();
-
-        if (getRawBytes() != null) {
-            final byte[] ret = new byte[length];
-            System.arraycopy(getRawBytes(), offset, ret, 0, length);
-
-            return ret;
-        }
-
-        byte[] leftBytes;
-        byte[] rightBytes;
-        final int leftLength = left.byteLength();
-
-        if (offset < leftLength) {
-            // The left branch might not be large enough to extract the full byte range we want. In that case,
```

Comment:
While it's the only comment here, I think carrying it forward in `RopeOperations` has merit.